proc/syscall_windows.go

```diff
@@ -22,8 +22,53 @@ type _THREAD_BASIC_INFORMATION struct {
 	BasePriority   int32
 }
 
+type _CREATE_PROCESS_DEBUG_INFO struct {
+	File                syscall.Handle
+	Process             syscall.Handle
+	Thread              syscall.Handle
+	BaseOfImage         uintptr
+	DebugInfoFileOffset uint16
+	DebugInfoSize       uint16
```

Comment:
`DebugInfoFileOffset` and `DebugInfoSize` are `DWORD`s, right?  https://msdn.microsoft.com/en-us/library/windows/desktop/ms679286(v=vs.85).aspx.  Delve isn't using anything in this struct past the `Thread` field, so likely this isn't causing any problems yet.